common/service/service.go

```diff
@@ -73,6 +72,7 @@ type (
 		DispatcherProvider  client.DispatcherProvider
 		BlobstoreClient     blobstore.Client
 		DCRedirectionPolicy config.DCRedirectionPolicy
+		PublicClient        workflowserviceclient.Interface
```

Comment:
do we use a different client in the internal repo?

Reply:
There will be a follow up diff in the internal repo. This diff needs more context. I can explain in person on monday.